service_container/autowiring.rst

```diff
@@ -635,11 +635,15 @@ The ``#[Autowire]`` attribute can also be used for :ref:`parameters <service-par
 
             // expressions
             #[Autowire(expression: 'service("App\\\Mail\\\MailerConfiguration").getMailerMethod()')]
-            string $mailerMethod
+            string $mailerMethod,
 
             // environment variables
             #[Autowire(env: 'SOME_ENV_VAR')]
-            string $senderName
+            string $senderName,
+
+            // environment variables with processors
+            #[Autowire(env: 'bool:SOME_BOOL_ENV_VAR')]
+            string $allowAttachments,
```

Comment:
```suggestion
            bool $allowAttachments,
```
?

Reply:
Fixed while merging

Reply:
Wow, what a silly mistake. Thank you very much!